modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/checkpoint/Checkpointer.java

```diff
@@ -616,6 +616,25 @@ private void updateMetrics(Checkpoint chp, CheckpointMetricsTracker tracker) {
                 tracker.cowPagesWritten()
             );
         }
+
+        if (cacheProcessor.context().kernalContext().performanceStatistics().enabled()) {
```

Comment:
`cacheProcessor.context().kernalContext().performanceStatistics()` let's cache this into instance variable

Reply:
fixed